_pages/support.md

```diff
@@ -392,4 +411,60 @@ SAML requests from browser consoles are URI encoded, base-64-encoded, and deflat
 </div>
 </div>
 
+<h4 class="usa-accordion__heading">
+<button class="usa-accordion__button" aria-controls="samlsignaturetroubleshooting">
+SAML Signature Troubleshooting
+</button>
+</h4>
+<div id="samlsignaturetroubleshooting" class="usa-accordion__container">
+<div class="usa-accordion__content" markdown="1"  aria-expanded="true"> 
+Login.gov uses the cryptographic signatures of authentication requests to determine which public certificate to use when encrypting data in the SAML response. If the signature is not present, or cannot be validated successfully, you will encounter problems when you rotate your application’s key pair.
+
+<b> Check signature is present </b>
+
+If you are not sure whether your application is currently signing authentication requests, the easiest way to check is through the network tab in your browser's developer tools. Look for the URL generated when your app sends the user to Login.gov's sign-in page.
+
+The signature could be sent in one of two ways:
+<div>
+<ol class="usa-list">
+      <li>As a parameter within the URL,</li>
+      <li>or as a field within the authentication request’s SAML data.</li>
+    </ol>
+  </div>
+
+If the <b>Signature</b> and <b>SigAlg</b> URL parameters (and associated values) are present, your authentication request is signed.
+If the signature is not part of the URL, it may be part of the SAML request. To check this, you will need to decode the data sent via the <b>SAMLRequest</b> parameter. The easiest way to do this is the "SAML Tracer" browser plugin. Our <a class="usa-link" href="http://dashboard.int.identitysandbox.gov/tools">a web-based tool</a> can also help with this. Once decoded, you should see a section that contains all the relevant signature-related information and should be enclosed in a tag like:
```

Comment:
```suggestion
If the signature is not part of the URL, it may be part of the SAML request. To check this, you will need to decode the data sent via the <b>SAMLRequest</b> parameter. The easiest way to do this is the "SAML Tracer" browser plugin. Our <a class="usa-link" href="http://dashboard.int.identitysandbox.gov/tools">web-based tool</a> can also help with this. Once decoded, you should see a section that contains all the relevant signature-related information and should be enclosed in a tag like:
```

Reply:
it doesn't look like this was fixed! it still says "Our a web-based tool</a>"